module-2/backend/week_three.md

```diff
@@ -10,16 +10,40 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What are the 3 main features in an ERD?
+Entities, attributes, relationships
 2. Create an ERD for the following database: Restaurants, Customers, Items, Ingredients, Beverages, Orders, Vendors.
+done
 3. What is the entry at the command line to create a new rails app?
+rails new (appname) -d (database)
 4. What do Models generally inherit from in rails?
+ActiveRecord::Base
 5. What do Controllers generally inherit from in a rails project?
+ApplicationController
 6. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+resources :horses, only: [:show]
 7. What rake task is useful when looking at routes, and what information does it give you?
+rake routes. Gives you prefix, verb, uri pattern, and controller#action.
 8. What is an example of a route helper? When would you use them?
+playlists_path. takes you to a specific page in your site.
 9. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+path just returns the domain, while url includes the entire url.
 10. What are strong params and why are the necessary?
+params that you have to make a decision on to allow for mass update.
```

Comment:
these are the params that we want to accept for mass update. they are necessary because we don't want malicious users to submit key/value pairs in our params 👍